src/main/java/com/mraof/minestuck/skaianet/Session.java

```diff
@@ -27,6 +28,8 @@
 	
 	final Map<PlayerIdentifier, PredefineData> predefinedPlayers;
 	final Set<SburbConnection> connections;
+	final GristGutter gutter;
+	private double gutterMultiplier;
```

Comment:
What is the difference between the gutter multiplier in the session and the gutter multiplier in player data?